### DISCUSSION TITLE
Line numbers included in answer

### DISCUSSION THREAD

**Discussion by Irubataru** [CopilotC-Nvim/CopilotChat.nvim#420] [Q&A] [answered]
Hi!

I am relatively new to the plugin, but it seems to be very useful. Thanks!

One issue I am having is that is sometimes (but not always) includes line numbers in the answer. If I then accept that by pressing `<C-y>`, the line numbers are pasted into the file, which obviously isn't valid syntax. Is there a way to prevent this?

![image](https://github.com/user-attachments/assets/a4173ed6-7b01-43e3-bb87-6a9f64178b31)

**Comment by CarlosHPlata** [comment]
I'm having the same issue, were you able to solve it?

**Comment by pidgeon777** [comment, marked as answer]
Using `claude-3.5-sonnet`, this issue seems not to occur.

**Reply by deathbeam** [reply]
Its mostly intermittent issue when the AI hallucinates. when providing context we provide line numbers as well but then tell AI to not return them in response, which 99% of the time works fine but sometimes it returns numbers anyway. But yea I think with claude the AI hallucinates a bit less so its a bit more reliable

**Comment by deathbeam** [comment]
For anyone experiencing this still, try `claude-3.5-sonnet` instead of default model, it behaves a lot better and you almost never see line numbers.
